### PR TITLE
Create java.md

### PR DESCRIPTION
**Pull request by maverick-27**

### CODE REVIEW

File: java.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Resources for JAVA
+
+## For Beginners
+
+1.  [Java For Beginner](https://www.youtube.com/watch?v=r59xYe3Vyks&list=PLS1QulWo1RIbfTjQvTdj8Y6yyq4R7g-Al)
+1.  [Learning References (GeeksForGeeks)](https://www.geeksforgeeks.org/java//)
+1.  [Code Monk](https://www.hackerearth.com/practice/)
```
Comment (saranshkataria) [inline java.md:7]: This is a generic programming resource, which does not belong here

File: java.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+1.  [Java2Blog](https://java2blog.com/)
+
+## For People with Intermediate Level Expertise And Web Editions
+
+1.  [Java Brains](https://www.youtube.com/user/koushks)
+1.  [Java2Blog](https://java2blog.com/)
```
Comment (saranshkataria) [inline java.md:13]: Put this only once, either in beginners or intermediate

File: java.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## For Experts
+
+1.  [Java For Experts](https://www.youtube.com/watch?v=vW53w7me4AE&list=PLAsTCriDN8nldLwhmPAUc7AzeiCLtnjEF)
```
Comment (saranshkataria) [inline java.md:17]: Looks like an intermediate resource

**Review by saranshkataria — CHANGES_REQUESTED** [review]
See inline comments. Also, add changes to README.md in this PR instead of creating a separate one

File: java.md (hunk 1 above)
Comment (thecreepaccess) [reply java.md]: Ok bhaiya  I will work on some stuff then update

**Comment by saranshkataria** [comment]
@thecreepaccess are you going to resolve the mentioned comments?

**Comment by saranshkataria** [comment]
closing due to inactivity

**Pull request closed without merging**